b2handle/tests/README.md

```diff
@@ -1,10 +1,18 @@
 # B2HANDLE Testing
 
+## Python 3 support
+
+As of version 1.1.0 the B2HANDLE library supports Python 3.
+Only for Python 3 users, the value of PYTHONHASHSEED variable should be set to 0 before running the tests.
+This can be simply performed by:
+
+    export PYTHONHASHSEED=0
+
 ## Testing with plain unittest/unittest2
 
 Simply run:
 
-    python main_test_script.py
+    python setup.py test
```

Comment:
Since running `python setup.up test` will invoke `nose` I would suggest to move this to the subsection below as an alternative method to run tests with `nose` and generate coverage reports (see also nosetests configuration in`setup.cfg`).